src/iconsManifest/supportedExtensions.ts

```diff
@@ -788,6 +788,12 @@ export const extensions: IFileCollection = {
       filename: true,
       format: FileFormat.svg,
     },
+    {
+      icon: 'bruno',
+      extensions: ['bru'],
```

Comment:
The extension is not needed given that you have provided a language id. This means that the Bruno extension will expose that id and set the proper extensions (I checked it out and that's already in place).

So basically, you have to decide what to do, either use language id or declare the extension.

Reply:
This is true, but leaving out `extension` means the icon would only be applied if the Bruno extension is installed. Leaving out `languages` means the icon is not applied to the `bruno` language in the language selector

Reply:
Yes, that's right. We have been historically not adding the extension if there's a language id already supporting it. This is basically because the extension has priority over the language id. So if the extension owner then decides that a particular extension is no supported anymore but we still have it,  we will have to actively remove it. The idea is that we wouldn't have to pay the price of maintaining the right extensions if possible.

Reply:
Also, we usually default to the language id in our decission process when that alternative exists

Reply:
It seems that the icon is not easily optimizable. Let's keep it as it is for the moment.